.gitlab-ci.yml

```diff
@@ -66,6 +66,7 @@ prepare-rhel-internal-runners:
       - RUNNER:
           # NOTE: 1 runner prepares for all arches b/c subsequent jobs download
           # artifacts from all previous jobs and the last one wins
+          - aws/rhel-9.6-nightly-x86_64
           - aws/rhel-10.0-nightly-x86_64
```

Comment:
@narmaku Nico, osbuild-composer has [two runners](https://github.com/osbuild/osbuild-composer/blob/main/.gitlab-ci.yml#L181) here. Is there any reason why we have/had only one?

Reply:
Yes, back in the days with Juan (former Image Builder team member) we discovered that you only need one runner to prepare the internals for all versions and architectures.

So, we reached to the conclusion that we don't really need both to run, just one. However this information is from ~2 years ago and may be wrong. Better to double-check with someone from osbuild. CC @thozza

Reply:
Ours `.gitlab-ci.yml` has different `rules` set for the job, see https://github.com/osbuild/osbuild-composer/blob/9b850de9f1f113744ad230420c2ce115f9afe610/.gitlab-ci.yml#L167-L168

As a result, only one runner is used, depending on the RHEL major version it runs for.
Examples:
9.6 nightly pipeline - https://gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/-/pipelines/1688676319
10.0 nightly pipeline - https://gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/-/pipelines/1688571275

So our case is different from yous, mainly because of the `rules`.